tests/CommandArtisanUITest.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+
+namespace Lorisleiva\ArtisanUI\Tests;
+
+
+use Lorisleiva\ArtisanUI\ArtisanUI;
+use Lorisleiva\ArtisanUI\Command;
+
+it('Accordion Closed when there are no arguments needed', function () {
```

Comment:
Thank you so much for adding tests!

Just a few comments regarding the naming.

- I usually name the test methods in a way that is uses the "it" provided by the function name. For example: `it('closes the accordion when there are no arguments')`. Alternatively, you use the `test(...)` method in PEST which doesn't prefix your test names with "it".
- I usually like to name PEST files like feature tests so I'd typically name this something more specific like `OpenArgumentsAccordionTest`.